.browserstack

```diff
@@ -0,0 +1,3 @@
+[credentials]
+username = BROWSERSTACK_USERNAME
+key = BROWSERSTACK_ACCESS_KEY
```

Comment:
EoF fixes on multiple files, should have an empty new line at the end of file.

Reply:
fixed https://github.com/browserstack/pytest-browserstack/pull/2/commits/da78880ffa43614c01ef958dfca28ba1dcdea32e